CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
-## [15-4.1] - 2024-09-06
-* Android 15
+## [15-5.0] - 2024-10-15
+* First Android 15 release
+* New backup format using compression and deduplication
+* Can still restore old backups, but old Seedvault can't restore backups from this version
+* Faster and more reliable backups making snapshots that can individually be restored
+* Auto-cleaning of old backups
+* D2D backups now default (circumvents apps not allowing backup)
```

Comment:
It can't be turned off, right? So I wouldn't say it's the default, but maybe something like "All backups are now D2D backups (circumvents apps not allowing backup)"

Reply:
updated, does it work like this?